src/transitions/Slide.spec.js

```diff
@@ -181,4 +182,37 @@ describe('<Slide />', () => {
       });
     });
   });
+
+  describe('slidemount', () => {
+    let mount;
+    let mountWrapper;
+    let instance;
+
+    before(() => {
+      mount = createMount();
+      mountWrapper = mount(<Slide><div /></Slide>);
+    });
+
+    after(() => {
+      mount.cleanUp();
+    });
+
+    it('default value', () => {
+      mountWrapper.setProps({ in: false });
+      instance = mountWrapper.instance();
+      instance.componentDidMount();
+      assert.strictEqual(
+        ReactDOM.findDOMNode(mountWrapper.instance().transition).style.transform,
```

Comment:
Yeah. Read the entire thread, both on the React repo and our internal issue. Not sure how to overcome this. What if `findDOMNode` returns Text here? Why would that happen? Should I throw an error then?